docs/guide/slots.md

```diff
@@ -175,6 +175,16 @@ class TableComponent < ViewComponent::Base
 end
 ```
 
+To provide content for a lambda slot via a block, add a block parameter and render the content by calling its `call` method:
```

Comment:
I don't think this text matches the example code any more 😄

Reply:
ooh good point! What about this?

```suggestion
To provide content for a lambda slot via a block, add a block parameter. You can then render the content by calling the block's `call` method, or pass the block directly to `content_tag`:
```